crates/e2e/tests/e2e/protocol_fee.rs

```diff
@@ -68,166 +38,7 @@ async fn local_node_price_improvement_fee_sell_order() {
-    // Market config could be used since the order is in-market.
-    let protocol_fee = ProtocolFee {
-        policy: fee_policy,
-        policy_order_class: FeePolicyOrderClass::Any,
-    };
-    // Without protocol fee:
-    // Expected execution is 10000000000000000000 GNO for
-    // 9871415430342266811 DAI, with executed_surplus_fee = 167058994203399 GNO
-    //
-    // With protocol fee:
-    // Expected executed_surplus_fee is 167058994203399 +
-    // 0.1*(10000000000000000000 - 167058994203399) = 1000150353094783059
-    //
-    // Final execution is 10000000000000000000 GNO for 8884273887308040129 DAI, with
-    // executed_surplus_fee = 1000150353094783059 GNO
-    //
-    // Settlement contract balance after execution = 1000150353094783059 GNO =
-    // 1000150353094783059 GNO * 8884273887308040129 / (10000000000000000000 -
-    // 1000150353094783059) = 987306456662572858 DAI
-    execute_test(
-        web3.clone(),
-        vec![ProtocolFeesConfig(vec![protocol_fee])],
-        OrderKind::Sell,
-        None,
-        1000150353094783059u128.into(),
-        987306456662572858u128.into(),
-    )
-    .await;
-}
-
 async fn partner_fee_sell_order_test(web3: Web3) {
-    // Fee policy to be overwritten by the partner fee + capped to 0.01
-    let fee_policy = FeePolicyKind::PriceImprovement {
-        factor: 0.5,
-        max_volume_factor: 0.9,
-    };
-    let protocol_fee = ProtocolFee {
-        policy: fee_policy,
-        policy_order_class: FeePolicyOrderClass::Market,
-    };
-    // Without protocol fee:
-    // Expected execution is 10000000000000000000 GNO for
-    // 9871415430342266811 DAI, with executed_surplus_fee = 167058994203399 GNO
-    //
-    // With protocol fee:
-    // Expected executed_surplus_fee is 167058994203399 +
-    // 0.01*(10000000000000000000 - 167058994203399) = 100165388404261365
-    //
-    // Final execution is 10000000000000000000 GNO for 9772701276038844388 DAI, with
-    // executed_surplus_fee = 100165388404261365 GNO
-    //
-    // Settlement contract balance after execution = 100165388404261365 GNO =
-    // 100165388404261365 GNO * 9772701276038844388 / (10000000000000000000 -
-    // 100165388404261365) = 98879067931768848 DAI
-    execute_test(
-        web3.clone(),
-        vec![ProtocolFeesConfig(vec![protocol_fee])],
-        OrderKind::Sell,
-        Some(OrderCreationAppData::Full {
-            full: json!({
-                "version": "1.1.0",
-                "metadata": {
-                    "partnerFee": {
-                        "bps":1000,
-                        "recipient": "0xb6BAd41ae76A11D10f7b0E664C5007b908bC77C9",
-                    }
-                }
-            })
-            .to_string(),
-        }),
-        100165388404261365u128.into(),
-        98879067931768848u128.into(),
-    )
-    .await;
-}
-
-async fn partner_fee_with_not_default_cap_sell_order_test(web3: Web3) {
```

Comment:
Is it intended that this fee still has a fee policy for market orders enabled?

Reply:
Partner fee should replace any configured protocol fee. Why not keep the Market fee here?

Reply:
> Partner fee should replace any configured protocol fee.

Well, not really. It's just an implementation detail right now that it does. I'd hope that we can support multiple fee policies very soon so I'm not sure we want to explicitly test for this yet (but also no strong feeling).

Reply:
We have this task for it: https://github.com/cowprotocol/services/issues/2495